main/src/cgeo/geocaching/MainActivity.java

```diff
@@ -198,11 +200,13 @@ private static String formatAddress(final Address address) {
         return StringUtils.join(addressParts, ", ");
     }
 
-    private final Consumer<GpsStatusProvider.Status> satellitesHandler = new Consumer<Status>() {
+    private final Consumer<GnssStatusProvider.Status> satellitesHandler = new Consumer<Status>() {
         @Override
-        public void accept(final Status gpsStatus) {
-            if (gpsStatus.gpsEnabled) {
-                navSatellites.setText(res.getString(R.string.loc_sat) + ": " + gpsStatus.satellitesFixed + '/' + gpsStatus.satellitesVisible);
+        public void accept(final Status gnssStatus) {
+            if (gnssStatus.gnssEnabled) {
+                navSatellites.setText(res.getString(R.string.loc_sat) + ": " + gnssStatus.satellitesFixed + '/' + gnssStatus.satellitesVisible);
+            } else if (VERSION.SDK_INT >= VERSION_CODES.N) {
+                navSatellites.setText(res.getString(R.string.loc_gnss_disabled));
```

Comment:
I'm not sure of this additional new status. I never heard of GNSS before, and I would guess the same for many other of our users. For those "GPS" is what they want, independent of what we use to calculate the status.
Basically, this is the general UX best practice "never use implementation terminology in the user interface, the user has a different mental model than the developer".

Reply:
Yes, lets leave the UI untouched.

 Basically all people only know GPS and still expect, that all other sat systems are used as well.